MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -60,11 +62,17 @@ public StreamBuilder(ITranscoderSupport transcoderSupport, ILogger logger)
                 if (streamInfo is not null)
                 {
                     streamInfo.DeviceId = options.DeviceId;
-                    streamInfo.DeviceProfileId = options.Profile.Id;
+                    streamInfo.DeviceProfileId = options.Profile.Id.ToString("N", CultureInfo.InvariantCulture);
                     streams.Add(streamInfo);
                 }
             }
 
+            foreach (var stream in streams)
```

Comment:
This seems a little pointless? streams is initially an empty list and is populated with StreamInfo instances that already have deviceid and profile id set.